### ISSUE TITLE
Cosmetics & Additional Cleanup

### ISSUE THREAD

**Issue by tvaroglu** [tvaroglu/spot_me_frontend#56]
In addition to adding new (& stretch) features, I would like to make sure what we have currently built is clean as a whistle.  List below of the items that I currently feel a personal need to address, so feel free to add additional ones and comment in if you'd like to revisit approach on anything I'm currently proposing at this point:

- [ ] Update `dashboard` and `profile` 'Add Friend' and 'Friends' section to maybe be `Add Follower` and `Followers` section.  Because the relationship is one-sided, it's a little confusing that you end up on a "friend's" profile and don't see yourself as a friend.  Regardless of whether we decide to forklift the back end logic and make the relationship two-sided (not my vote at this point, would prefer to focus on other stretch features first), I think this simple fix makes sense for the near term.
- [ ] Comb back through any of the flash messages (overall app branding) for how we're communicating to our users (i.e. have noticed `swol-mate` vs `swole-mate` in certain places, ReadMe included
- [ ] Double-check feature testing to see if we can slim down anything (stub-wise) to consolidate the logic a little bit.  It was a mad dash at the end, and there are probably extra stubs that we may or may not need, and at this point, it's impossible to tell without doing a thorough once over.  If we have slimmed down feature testing to the bare bones, we can implement additional features much quicker, as we'll know exactly how the flow needs to go.
- [ ] Along the lines of the prior point re: testing file refactoring / review, we should check coverage report to see if there are any additional un-tested methods (or sad paths) that we can test.  Won't be possible to get to 100%, but anything that we might have missed and can reasonably "test", I'd like to try, even if only at a very basic level.

**Comment by tvaroglu** [comment]
FYI @Scott-Borecki , please let me know your thoughts when you've had a chance to take a look.

**Comment by Scott-Borecki** [comment]
> Update dashboard and profile 'Add Friend' and 'Friends' section to maybe be Add Follower and Followers section. Because the relationship is one-sided, it's a little confusing that you end up on a "friend's" profile and don't see yourself as a friend. Regardless of whether we decide to forklift the back end logic and make the relationship two-sided (not my vote at this point, would prefer to focus on other stretch features first), I think this simple fix makes sense for the near term.

I agree we should clear this up.  I love the idea of having both `Following` and `Followers` sections on the dashboard/profiles.  That would allow a user to see who is following them and whether they want to follow back and schedule a workout with them.

**Comment by Scott-Borecki** [comment]
> Comb back through any of the flash messages (overall app branding) for how we're communicating to our users (i.e. have noticed swol-mate vs swole-mate in certain places, ReadMe included)

Agreed on this.  I think most important for democomp will be the presentation/visual aesthetic of the website and telling a good story around it.  The code definitely needs to work, but we'll have the best chances if our website display is consistent and clean.

**Comment by Scott-Borecki** [comment]
> Double-check feature testing to see if we can slim down anything (stub-wise) to consolidate the logic a little bit.

Me being me, I would love to refactor.  We will just need to get everyone's buy-in on refactoring and I would love to pair with anyone that would want to tackle that with me.

**Comment by Scott-Borecki** [comment]
> Along the lines of the prior point re: testing file refactoring / review

I think there is definitely some low-hanging fruit as far as backfilling the testing.  I probably wouldn't spend too much time messing around with challenging tests ... but more so backfilling the "easier" tests that we just didn't had time to write in the dash to the finish line.  For example, poros, serializers, etc.

**Comment by Scott-Borecki** [comment]
> Refactor routes file to look a little more restful via resources keywords

I think this should be a pretty "easy" lift.
